Instantiation

ghost opened the pull request:
Hey,

I've did a major rewrite of your really nice countdown timer. It should also work with the current specification.

Example for illustration:

```
var timer = $('#timer').final_countdown({
    seconds: {
        borderColor: '#fff',
        borderWidth: '10'
    },
    minutes: {
        borderColor: '#fff',
        borderWidth: '10'
    },
    hours: {
        borderColor: '#fff',
        borderWidth: '10'
    }
});

// Set times (start, end, now)
timer.set(new Date.now().getTime() - (15 * 1000), new Date.now().getTime() + (15 * 1000), new Date.now().getTime());

// Start the timer
timer.start();

// Stop the timer
timer.stop();
```

The improvements in detail:
- returns instance of countdown object
- counters aren't bind anymore to an id
- redefines (set) the timers dynamically
- start/stop timers
- arbitrary definition of days, hours, minutes and seconds

Improvements:
- "use strict" directive
- JSLinted

Solves:
PragmaticMates/jquery-final-countdown#5

Commit:
final_countdown instantiation:

- returns instance of countdown object
- counters aren't bind anymore to an id
- redefines (set) the timers dynamically
- start/stop timers
- arbitrary definition of days, hours, minutes and seconds

Improvements:
- "use strict" directive
- JSLinted

gbrusella commented:
Looks very nice indeed!! (and much easier usage).

zusatzstoff commented:
Thanks, but unfortunately there isn't any activity in this repo anymore. Hopefully the owners will merge this someday or I will have to empower the fork.

css-wizard commented:
As I undestand this allows to use multiple counters on the same page. Nice!
Can you show sample HTML? I've moved those IDs to classes but I still doesn't work for me.

zusatzstoff commented:
Have a look at my fork, I also adjusted the readme: https://github.com/zusatzstoff/jquery-final-countdown

A live example can be seen here: http://zusatzstoff.github.io/guild-wars-2-timer

css-wizard commented:
Thank you, it helped alot. I had to edit some bits and bobs in JS file in order to make all the circles work.

zusatzstoff commented:
Could you create a PR with your changes on my fork?

wojciech-trybula committed:
Update jquery.final-countdown.js

I fixed some minor typos and circles calculations as it didn't work properly prior to these changes.

Commit:
Merge pull request #1 from css-wizard/patch-1

Update jquery.final-countdown.js
